src/views/v2/rsvp/actions/rsvp/not-going.php

```diff
@@ -40,6 +40,6 @@
 		class="tribe-common-cta tribe-common-cta--alt tribe-tickets__rsvp-actions-button-not-going"
 		<?php tribe_disabled( $must_login ); ?>
 	>
-		<?php esc_html_e( "Can't go", 'event-tickets' ); ?>
+		<?php echo esc_html_x( "Can't go", 'Label for the RSVP "can't go" version of the not going button', 'event-tickets' ); ?>
```

Comment:
* :no_entry_sign: PHP syntax error: syntax error, unexpected 't' (T_STRING), expecting ',' or ')'
* :no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: 046be24cd71477823f1d6ca1853f41a2</sub>

Reply:
@sc0ttkclark escape quote

Reply:
```suggestion
		<?php echo esc_html_x( "Can't go", 'Label for the RSVP "can\'t go" version of the not going button', 'event-tickets' ); ?>
```